geoconvert/data/subdivisions/germany.py

```diff
@@ -45,6 +45,80 @@
     "erfurt": "TH",
 }
 
+# source https://simple.wikipedia.org/wiki/Postal_codes_in_Germany
+DE_POSTCODE_RANGE = {
```

Comment:
Do you need another thorough look at that data? In other words, are you sure you took care of all special cases here?